### PR TITLE
Main7

### PR DESCRIPTION
**Pull request by Meseret-Melkamu**
approve my request please

### CODE REVIEW

**Review by mohammedbedru — APPROVED** [review]

**Review by mohammedbedru — APPROVED** [review]

File: ui/src/app/app-routing.module.ts
Comment (Meseret-Melkamu) [inline ui/src/app/app-routing.module.ts:1]: need to remove the application list files

File: api/server.js
Comment (Meseret-Melkamu) [inline api/server.js:1]: remove engineering valuation title of server

File: ui/src/app/app.component.html
Comment (Meseret-Melkamu) [inline ui/src/app/app.component.html:1]: application router link not removed

File: ui/src/app/app.module.ts
Comment (Meseret-Melkamu) [inline ui/src/app/app.module.ts:1]: application modules not removed

File: ui/src/app/app/pages/add-rght/add-rght.component.spec.ts
Comment (Meseret-Melkamu) [inline ui/src/app/app/pages/add-rght/add-rght.component.spec.ts:1]: remove application files module added

**Pull request closed without merging**